packages/pyelftools/build.sh

```
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -ex
+
+pip3 wheel -w dist --no-build-isolation --no-deps --no-cache-dir $(pwd)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)